source/imgui/CMakeLists.txt

```diff
@@ -42,7 +42,11 @@ file(GLOB FREETYPE_HEADERS
   "${IMGUI_ROOT}/misc/freetype/*.h"
 )
 
-add_library(imgui SHARED ${SOURCES} ${HEADERS} ${BACKEND_HEADERS} ${FREETYPE_HEADERS})
+file(GLOB MISC_HEADERS
+  "${IMGUI_ROOT}/misc/cpp/imgui_stdlib.h"
+)
```

Comment:
should we take all headers from misc folder?

Reply:
Fixed.